rootfs/python-build/Dockerfile

```diff
@@ -0,0 +1,65 @@
+# https://github.com/revsys/optimized-python-docker/blob/master/Dockerfile
+# https://github.com/docker-library/python/blob/master/3.12/slim-bullseye/Dockerfile
+
+FROM ubuntu:22.04 as base
+
+ENV PATH /usr/local/bin:$PATH
+
+RUN apt update && \
+	apt install -yqq build-essential && \
+	apt install -yqq make && \
+	apt install -yqq curl && \
+	apt install -yqq tar && \
+	apt install -yqq libexpat-dev && \
+	apt install -yqq zlib1g-dev && \
+	apt install -yqq zlib1g-dev
+
+# apt install -yqq libsqlite3-dev && \
+# apt install -yqq libexpat1-dev && \
+# apt install -yqq libffi-dev
+
+ARG PYTHON_VERSION
+ENV PYTHON_VERSION ${PYTHON_VERSION}
+
+ENV SRCDIR /python
+RUN mkdir -p ${SRCDIR}/build && \
+	curl -sL "https://www.python.org/ftp/python/${PYTHON_VERSION}/Python-${PYTHON_VERSION}.tar.xz" | tar -xJC ${SRCDIR} --strip-components=1 -f -
```

Comment:
Prefix `RUN` commands with `set -xe;` and prefer `;` over `&&`

Reply:
`&&` is required in case you want to run the second command only if the first command completed successfully. I added `set -xe`.

Reply:
that's what `set -xe` enables, `&&` is no longer necessary with `set -xe`